example/test-make-vcow-swappable.json

```diff
@@ -0,0 +1,8 @@
+{
+  "mainnet": {
```

Comment:
To me the grouping is handy as mainnet contracts will be written as `mainnet.vCow`, compared to `mainnetVcow`. So the choice is between:

```
{
"mainnet": {
    "cowToken": "0xDEf1CA1fb7FBcDC777520aa7f396b4E015F497aB",
    "virtualCowToken": "0xd057b63f5e69cf1b929b356b579cba08d7688048",
    "atoms": "123456000000000000000000",
    "multisend": "0x8d29be29923b68abfdd21e541b9374737b49cdad"
  },
"gnosisChain": {
    "cowToken": "0xDEf1CA1fb7FBcDC777520aa7f396b4E015F497aB",
    "virtualCowToken": "0xd057b63f5e69cf1b929b356b579cba08d7688048",
    "atoms": "123456000000000000000000",
    "multisend": "0x8d29be29923b68abfdd21e541b9374737b49cdad"
  }
}
```

and:

```
{
  "mainnetCowToken": "0xDEf1CA1fb7FBcDC777520aa7f396b4E015F497aB",
  "mainnetVirtualCowToken": "0xd057b63f5e69cf1b929b356b579cba08d7688048",
  "mainnetAtoms": "123456000000000000000000",
  "mainnetMultisend": "0x8d29be29923b68abfdd21e541b9374737b49cdad"
  "gcCowToken": "0xDEf1CA1fb7FBcDC777520aa7f396b4E015F497aB",
  "gcVirtualCowToken": "0xd057b63f5e69cf1b929b356b579cba08d7688048",
  "gcAtoms": "123456000000000000000000",
}
```

I personally prefer the first, and I think it might be better to clearly distinguish mainnet and GC, but I'm also ok with the second.

Reply:
yes, the grouping is valuable. I was thinking about such a config: 
```
{
  "VcowContractOnMainnet": {
      "virtualCowToken": "0xd057b63f5e69cf1b929b356b579cba08d7688048",
      "atomsToTransfer": "123456000000000000000000",
    },
  "VcowContractOnGC": {
      "virtualCowToken": "different address",
      "atomsToTransfer": "123456000000000000000000",
    }
    "cowToken": "0xDEf1CA1fb7FBcDC777520aa7f396b4E015F497aB",
    "multisend": "0x8d29be29923b68abfdd21e541b9374737b49cdad"
}
```
Sorry for not making it more clear before.

But I am also totally with your current version. Feel free to ignore all my suggestions

Reply:
I see, at first I did the splitting in two because I want to clearly point out that also the multisig and the cow token belong to mainnet, but maybe that's not really helpful as you mention.

For this PR, in hindsight I shouldn't have included "mainnet" as there is only mainnet available so it should be implied. So for now I just removed the `mainnet` wrapper (and renamed `atoms` to `atomsToTransfer`). In this way this discussion should not be blocking this PR. Anyway I'm soon publishing the PR for the bridging on GC, and I think this discussion would be more fitting there.

On the topic, I'm wondering if actually using `mainnet` and `GC` is actually a good thing, since in a way it hardcodes the networks we want to use. I'm now thinking about something like this now:

```
{
  "bridged": {
    "virtualCowToken": "0xd057b63f5e69cf1b929b356b579cba08d7688048",
    "atomsToTransfer": "123456000000000000000000",
  },
  "virtualCowToken": "different address",
  "atomsToTransfer": "123456000000000000000000",
  "cowToken": "0xDEf1CA1fb7FBcDC777520aa7f396b4E015F497aB",
  "multisend": "0x8d29be29923b68abfdd21e541b9374737b49cdad"
  "multiTokenMediator": "0x88ad09518695c6c3712AC10a214bE5109a655671",
}
```
The implication being that the lower level is the network where the script is run, while the `bridged` level is the network we bridge to.

Reply:
You are right. This way it makes the most sense!